cco-d-acts.ttl

```diff
@@ -8,19 +7,21 @@
                                                dc:contributor "Jonathan M. Vajda" ;
                                                dc:license <https://creativecommons.org/licenses/by/4.0/> ;
-                                               rdfs:comment "CCO-D-Acts fills a gap in representation of social, legal, normative, and deontic entities for ontologies that rely on the Common Core Ontologies (CCO) suite (https://github.com/CommonCoreOntology/CommonCoreOntologies). Within the Basic Formal Ontology (BFO) (https://github.com/bfo-ontology/) ecosystem, the Document Acts (D-Acts) ontology (https://github.com/d-acts/d-acts) is an extension of the Information Artifact Ontology (IAO) that represents the generic domain of entities relevant to the creation, modification, or termination of social arrangements by means of some document. Examples of such documents include marriage licenses, mortgage contracts, parking tickets, etc. The Informed Consent Ontology (ICO) imports D-Acts, for example. The Common Core Ontologies (CCO) are a suite of mid-level ontologies conformant with BFO, that provide representation of generic domains such as agents, artifacts, events, information, etc. The Information Entity Ontology (IEO) is one of those mid-level ontologies of the CCO suite, with a high overlap in domain as IAO. However, IAO and IEO have conflicting models. Since D-Acts is an extension of IAO, it would benefit those who rely on CCO ontologies to have a D-Acts counterpart conformant with IEO. This project proposes that counterpart: CCO-D-Acts."@en ,
-                                                            "The most recent version of this file is always hosted on GitHub https://github.com/jonathanvajda/cco-d-acts"@en .
+                                               dcterms:description "The Common Core Ontologies (CCO) Conformant Documents Ontology is a domain ontology that represents the generic domain of entities relevant to the creation, modification, or termination of social arrangements by means of some document. This includes social acts, deontic roles, and directive information and information bearing entities."@en ;
+                                               owl:versionIRI <https://github.com/jonathanvajda/2024-10-22/cco-d-acts> ;
+                                               owl:versionInfo "2024-10-22"@en ; 
+                                               rdfs:comment "CCO-D-Acts is designed to fill a gap in representation of social, legal, normative, and deontic entities for ontologies that rely on the Common Core Ontologies (CCO) suite (https://github.com/CommonCoreOntology/CommonCoreOntologies). Within the Basic Formal Ontology (BFO) ecosystem, the Document Acts (D-Acts) ontology (https://github.com/d-acts/d-acts) is an extension of the OBO Foundry Information Artifact Ontology (IAO). This ontology proposes to transposes that OBO ontology into a CCO equivalent."@en ,

```

Comment:
"This ontology proposes to transposes" I think transposes should be replaced with transpose